spec/requests/api/v1/addresses/create_spec.rb

```diff
@@ -0,0 +1,57 @@
+describe 'POST /api/v1/user/addresses', type: :request do
+  subject { post api_v1_user_addresses_path, headers: headers, params: params, as: :json }
+
+  let(:headers) { auth_headers }
+  let(:user) { create(:user) }
+  let(:city) { create(:city) }
+  let(:home_address) { Faker::Address.street_address }
+  let(:zip_code) { Faker::Address.zip_code }
+  let(:params) do
+    {
+      address: {
+        home_address: home_address,
+        zip_code: zip_code,
+        city_id: city.id
+      }
+    }
+  end
+
+  before { subject }
```

Comment:
I have the same suggestions I did for [this PR](https://github.com/szcrubyroostrap/rs-blackmarket/pull/15) regarding `before` blocks and missing `change` expectations